### PR TITLE
Service based groundedness evaluator

### PR DESCRIPTION
**Pull request by singankit**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by nagkumar91 — APPROVED** [review]

File: sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/evaluators/_groundedness/__init__.py [hunk 1]
```diff
@@ -2,8 +2,10 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-from ._groundedness import GroundednessEvaluator
+from ._groundedness import LLMGroundednessEvaluator
+from ._service_groundedness import GroundednessEvaluator
 
 __all__ = [
+    "LLMGroundednessEvaluator",
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/evaluators/_groundedness/__init__.py:9]: Not LlmGroundednessEvaluator?

File: sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/constants.py [hunk 2]
```diff
@@ -54,6 +55,7 @@ class EvaluationMetrics:
     SEXUAL = "sexual"
     PROTECTED_MATERIAL = "protected_material"
     XPIA = "xpia"
+    GROUNDEDNESS = "generic_groundedness"
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/constants.py:58]: Why not just call it "groundedness"? The other one is called "gpt_groundedness" which reflects it's a llm based.

File: sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_evaluate.py [hunk 3]
```diff
@@ -90,18 +91,25 @@ def _get_run_from_run_history(flow_run_id, ml_client, project_scope):
 @pytest.mark.usefixtures("recording_injection", "recorded_test")
 @pytest.mark.localtest
 class TestEvaluate:
-    @pytest.mark.skip(reason="Temporary skip to merge 37201, will re-enable in subsequent pr")
-    def test_evaluate_with_groundedness_evaluator(self, model_config, data_file):
+    # @pytest.mark.skip(reason="Temporary skip to merge 37201, will re-enable in subsequent pr")
+    def test_evaluate_with_groundedness_evaluator(self, model_config, project_scope, data_file):
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_evaluate.py:95]: please add an e2e test to test_builtin_evaluators.py as well.

File: sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/constants.py (hunk 2 above)
Comment (diondrapeck) [reply sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/constants.py:58]: The other one's name may be being changed. There's a spec to remove the "gpt_" prefix. I'll message it to you on Teams.

**Pull request closed without merging**